pkg/deploy/dev-workspace/dev_workspace_test.go

```diff
@@ -435,6 +435,47 @@ func TestShouldSyncObjectIfItWasCreatedBySameOriginHashDifferent(t *testing.T) {
 	}
 }
 
+func TestShouldNotSyncObjectIfItIsNotMangedByChe(t *testing.T) {
```

Comment:
I encourage to use asserts from the `github.com/stretchr/testify/assert` package for new tests.

Reply:
I'll provide a separate PR with adapting all suites in that file